ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```diff
@@ -12,6 +12,12 @@ export PATH=/usr/local/go/bin:/usr/libexec/origin:/opt/OpenShift4-tools:$PATH
 export REPORT_HANDLE_PATH="/usr/bin"
 export ENABLE_PRINT_EVENT_STDOUT=true
 
+# add for hosted kubeconfig in the hosted cluster env
```

Comment:
@heliubj18 please add this to the following step too.
openshift-extended-test-longduration
openshift-extended-test-disruptive

I am not sure if the following also need it:
openshift-extended-upgrade-pre
openshift-extended-upgrade-post

I think it has no side effect to add them into these two files.

Reply:
openshift-extended-upgrade-pre
openshift-extended-upgrade-post
are also required for HyperShift

Reply:
so, the more 4 steps need it too.

Reply:
got it, thanks